patches/tModLoader/Terraria/ModLoader/Default/UnloadedAccessorySlot.cs

```diff
@@ -0,0 +1,25 @@
+namespace Terraria.ModLoader.Default
+{
+	[Autoload(false)]
+	public class UnloadedAccessorySlot : ModAccessorySlot
+	{
+		public override string Name { get; }
+
+		internal UnloadedAccessorySlot(int slot, string oldName) {
+			type = slot;
+			Name = oldName;
+		}
+
+		public override bool IsSlotValid() { 
+			return false;
+		}
+
+		// Only display unloaded slots if in singleplayer. Don't need people carrying items in to multiplayer they shouldn't have :(
```

Comment:
unsure about this, may cause confusion. The item can switch to 'unloaded item' independently of the slot. I think if a server removes a mod, or a player joins a server which doesn't have the mod adding the slot, they should still be able to get the item and choose how/if to use it with the other slots they have available here.

Reply:
Gonna make a judgement call - if a player has an op accessory going in, odds are they have other op items elsewhere on the player.
This kind of check would thus be redundant.
Will default back to just if there is an item